try{
    hello();
} catch {
    console.log(error);
}

try{
    hello();
} catch {
    console.log('Esto es un error');
}
